js/ui/diagram/diagram.importer.js

```diff
@@ -1,12 +1,12 @@
-const Errors = require('../widget/ui.errors');
-const Diagram = require('devexpress-diagram');
+import Errors from '../widget/ui.errors';
+import * as Diagram from 'devexpress-diagram';
 
 function getDiagram() {
-    if(!Diagram) {
+    if(!Diagram.DiagramCommand) {
```

Comment:
@Romzay 
Is it a proper time to add a default export for this package (like [devexpress-gantt](https://github.com/DevExpress/DevExtreme/blob/caf26bfd208df804af8a155565bff646399073bc/js/ui/gantt/gantt_importer.js))?

So we can avoid of using specific fields to check if package imported.

```js
import Diagram from 'devexpress-diagram'
```

Reply:
FYI: discussed it with Roman, we'll fix it bit later